boost-maven/boost-runtimes/runtime-openliberty/src/main/java/org/microshed/boost/runtimes/openliberty/LibertyRuntime.java

```diff
@@ -79,6 +81,7 @@ public LibertyRuntime(RuntimeParams runtimeParams) {
         this.projectBuildDir = project.getBuild().getDirectory();
         this.libertyServerPath = projectBuildDir + "/liberty/wlp/usr/servers/" + serverName;
         this.mavenDepPlugin = runtimeParams.getMavenDepPlugin();
+        this.runtimeVersion = boostProperties.getProperty("boost_runtime_version", defaultRuntimeVersion);
```

Comment:
I'd lean towards using the same property as LMP:    -DlibertyRuntimeVersion.    For one..one less property name in the ecosystem.   Also... this isn't going to go into server config